lib/cisco_node_utils/cmd_ref/feature.yaml

```diff
@@ -1,5 +1,17 @@
 # feature
 ---
+fabric:
+  _exclude: [/N3/]
+  config_get: "show feature-set"
+  config_get_token: '/^fabric[\s\d]+(\w+)/'
+  config_set: "<state> feature-set fabric"
```

Comment:
Hardcode <state> as install, since we only install/enable, never uninstall/disable